File: 01_Programming_Basics/01_Programming_Basics_with_C++/06_Nested_Conditional_Statements_Lab/01_Personal_Titles.cpp
#include <iostream>
using namespace std;

int main() {
  double age = 0;
  string gender = "";
  string msg = "";

  cin >> age >> gender;

  if (gender == "m") {
    if (age < 16) {
      msg = "Master";
    } else {
      msg = "Mr.";
    }
  } else if (gender =="f") {
    if (age < 16) {
      msg = "Miss";
    } else {
      msg = "Ms.";
    }
  }

  cout << msg << endl;

  return 0;

}

File: 01_Programming_Basics/03_Programming_Basics_with_JavaScript/04_Nested_Conditional_Statements/01_PersonalTitles.js
function PersonalTitles(args) {
  let age = parseFloat(args[0]);
  let gender = args[1];
  let msg = '';

  if (gender == "m") {
    if (age >= 16) { msg = "Mr."; }
    else if (age >= 0 && age < 16) { msg = "Master"; }
  } else if (gender == "f") {
    if (age >= 16) { msg = "Ms."; }
    else if (age >= 0 && age < 16) { msg = "Miss"; }
  }

  console.log(msg);
}